wp-content/themes/missilethreat/archive.php

```diff
@@ -118,6 +94,11 @@
         
       <?php
 
+      $total_posts = $wp_the_query->found_posts;
+      $page = (get_query_var('paged')) ? get_query_var('paged') : 1;
+      $pages = $wp_the_query->max_num_pages;
+      echo '<h3 class="archive__results">' . $total_posts . ' items, Page ' . $page . ' of ' . $pages . '</h3>';
```

Comment:
This would be an `<h2>`, not a an `<h3>`. Remember, the headings have to occur in hierarchical order, so if it isn't nested below an `<h2>`, you can't use an h3.